starknet_py/net/client_models.py

```diff
@@ -752,18 +752,26 @@ class EntryPointsByType:
 
 
 @dataclass
-class DeprecatedContractClass:
+class _DeprecatedContract:
     """
     Dataclass representing contract declared to Starknet.
     """
 
     program: dict
     entry_points_by_type: EntryPointsByType
+
+
+@dataclass
+class DeprecatedContractClass(_DeprecatedContract):
+    """
+    Dataclass representing contract declared to Starknet.
```

Comment:
nit: Seems this one is more correct 😅
```suggestion
    Dataclass representing contract declared on Starknet.
```
<img width="347" alt="image" src="https://github.com/user-attachments/assets/cc4adb37-b499-49a3-8e52-1a3cc1c0e63e">